### PR TITLE
Implement remaining CSS and JS

### PR DESCRIPTION
**Pull request by philipstout3**

### CODE REVIEW

File: client/Stars.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+`
+
+class Stars extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return(
+      // TODO: CREATE STARS WITHOUT EXTERNAL LIB, ACCOUNT FOR RATINGS LOWER THAN 3
```
Comment (gakittle) [inline client/Stars.jsx:45]: This can be cleaned up, along with the commented-out block down below.

File: client/App.jsx [hunk 2]
```diff
@@ -26,44 +36,116 @@ const Texts = styled.div`
...
+            <Texts onClick={this.handleClick} id="twitterClicked" style={{color: (this.state.twitterClicked ? "#da3743" : null),  borderBottom: (this.state.twitterClicked ? "2px solid #da3743": null)}}>
+              Twitter
             </Texts>
           </Border>
-        </div>
+        <ParentBorder>
+          <Title restaurant={this.state.restaurant}/>
+          <Stars restaurant={this.state.restaurant}/>
+        </ParentBorder>
+        {/* CODE BELOW SHOULD BE INGORED -- ONLY USED FOR SCROLL TESTING */}
```
Comment (gakittle) [inline client/App.jsx:158]: Should this still be here?

**Review by gakittle — APPROVED** [review]
Great progress. I'd recommend setting up the AirBnb linter sooner rather than later. That'll probably break up many of the long lines where react components have a lot of props passed in.

File: client/Stars.jsx (hunk 1 above)
Comment (philipstout3) [reply client/Stars.jsx:45]: Good catch!

File: client/App.jsx (hunk 2 above)
Comment (philipstout3) [reply client/App.jsx:158]: No, this was for testing the automatic scroll on click

**Pull request merged**